CHANGELOG.asciidoc

```diff
@@ -31,6 +195,7 @@ https://github.com/elastic/beats/compare/v7.15.1...v7.15.2[View commits]
 - Remove unnecessary escaping step in dashboard loading, so they can be displayed in Kibana. {pull}28395[28395]
 - Fix AWS proxy_url config from url to string type.  {pull}28725[28725]
 - Fix `fingerprint` processor to give it access to the `@timestamp` field. {issue}28683[28683]
+- Skip `add_kubernetes_metadata` processor when Kubernetes metadata are already present {pull}27689[27689]
```

Comment:
This looks suspicious as it changes the 7.15.2 changelog? Maybe correct.

Reply:
yep, will fix in a follow up starting in 7.16